canonical/ght
# Temporarily Remove Cities Causing Geocoding Issues

## Rationale
- The geocoding API calls in GHT use only the city name. This causes the incorrect city to be returned in the cases where multiple cities share the same name. In order to fix this root issue, the location strings in GHT will have to be renamed and TS is currently assessing the best way to approach this.
- In the meantime however, they have requested that we remove both “Sudbury” and “St. John’s” from the list as they are repeat offenders of this issue and are not largely used at the moment anyway.
- Have also added Manila and Kuala Lumpur as new locations to GHT, as per request from TS.

## Done
- Temporarily removed "Sudbury" and "St. John's" from the regions list.

## QA
- Checkout this branch
- Install dependencies with `yarn install`
- Run GHT replicate command `yarn dev replicate -i`
- Ensure that Sudbury nor St. John's appear as region options

## Issue/Card
[WD-11485](https://warthogs.atlassian.net/browse/WD-11485), [WD-11487](https://warthogs.atlassian.net/browse/WD-11487)


[WD-11485]: https://warthogs.atlassian.net/browse/WD-11485?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[WD-11487]: https://warthogs.atlassian.net/browse/WD-11487?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
